website/docs/d/role_v2.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_role_v2"
+description: |-
+  Provides a way to retrieve Sumo Logic role details (id, names, etc) for a role managed outside of terraform.
+---
+
+# sumologic_role_v2 ( beta )
+
+Provides a way to retrieve Sumo Logic role details (id, names, etc) for a role
+managed by another terraform stack.
+
+
+## Example Usage
+```hcl
+data "sumologic_role_v2" "this" {
+  name = "MyRole"
+}
+```
+
+```hcl
+data "sumologic_role_v2" "that" {
+  id = "1234567890"
+}
+```
+
+A role can be looked up by either `id` or `name`. One of those attributes needs to be specified.
+
+If both `id` and `name` have been specified, `id` takes precedence.
```

Comment:
If both are specified, and they disagree (the only case in which precedence matters), is that not an error?

Reply:
This is the behaviour that's being used by role v1 data source, so am continuing with the same pattern right now. We can change this in future if needed ( IIUC you are prompting to throw an error if both are specified and they point to different responses , sounds fair tbh but since we would need this change in both v1 and v2 role datasource, am keeping this for a future change, not part of the current pr )